packages/cli/src/constants.ts

```diff
@@ -79,10 +80,10 @@ export const enum LICENSE_FEATURES {
 	API_DISABLED = 'feat:apiDisabled',
 }
 
-export const enum LICENSE_QUOTAS {
-	TRIGGER_LIMIT = 'quota:activeWorkflows',
-	VARIABLES_LIMIT = 'quota:maxVariables',
-	USERS_LIMIT = 'quota:users',
-}
+export const LICENSE_QUOTAS = {
+	TRIGGER_LIMIT: 'quota:activeWorkflows',
+	VARIABLES_LIMIT: 'quota:maxVariables',
+	USERS_LIMIT: 'quota:users',
+} as const;
```

Comment:
Love this! `LICENSE_FEATURES` should follow suit so we can combine them - this is why the typings were mismatching during our call. I'll submit a quick PR with some typings suggestions.